Add AYIM+MCA 2019 revival newspost (#5706)
ppy/osu-wiki: news/2021-06-14-mca-ayim-2019.md

Writing comments will be open **until** the results of MCA are posted.

All of these statistics and features can be found on the [AYIM Website](ayim.corsace.io)

## Mapper's Choice Awards 2019

MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards. 

Eligible users are those who ranked a beatmap, guest difficulty, or storyboard in 2019.

MCA has three stages: Nomination, voting, and results.

- **Nomination:** Eligible users can submit any maps/storyboards or users from 2019 they feel are deserving of recognition in their respective categories.
- **Voting:** Eligible users can vote on the pool of submitted nominations.
- **Results:** Will be based off the results of voting and feature a livestream discussing each of the top winners. 

This event will give perspective on what the community of that time finds to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019!

> Nomination and voting will be happening through the [MCA Website](mca.corsace.io), so be sure to check it out.
> ---
> We hope everyone can enjoy checking out MCA and AYIM as the events go on! Thank you to everyone who has made these possible, especially our main organizer [VINXIS](https://osu.ppy.sh/users/4323406). We look forward to seeing everyone's votes and comments for 2019.

Reviewer: ```suggestion
Nomination and voting will be happening through the [MCA Website](mca.corsace.io), so be sure to check it out.

---

We hope everyone can enjoy checking out MCA and AYIM as the events go on! Thank you to everyone who has made these possible, especially our main organizer [VINXIS](https://osu.ppy.sh/users/4323406). We look forward to seeing everyone's votes and comments for 2019.
```

---
layout: post
title: Mapper's Choice Awards and A Year In Mapping 2019
date: 2021-06-14 00:00:00 +0000
---

We're going to have a back to back experience for celebrating the 2019 and 2020 years of mapping!

> This June and July will be Mappers' Choice Awards _(MCA)_ and A Year in Mapping _(AYIM)_ for the 2019 season, and soon after that we will be hosting the same events but for 2020!

Reviewer 2: ```suggestion
This June and July will be Mappers' Choice Awards _(MCA)_ and A Year in Mapping _(AYIM)_ for the 2019 season! Soon after that, we will be hosting the same events but for 2020!
```

> It's been a while, but we're back with an epic pair of websites to allow for a better experience than ever.

Reviewer 2: ```suggestion
It's been a while, but we're also back with an epic pair of websites as well to allow for a better experience than ever!
```

For now, we're all about celebrating 2019!

## Schedule for MCA and AYIM 2019

| Event | Timestamp (UTC) |
| --: | :-- |
| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
| Results Stream | 2021-07-10 / 2021-07-11 |
| Results posted to Website | 2021-07-12 00:00 |

## A Year In Mapping 2019

> AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most maps, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and a whole lot more records than you can think of.

Reviewer 2: ```suggestion
AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and a whole lot more records than you can think of.
```

> **Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to the whole community. Know some banger maps from 2019 or a mapper left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open to _everyone_ to participate in, so let your positivty be heard!

Reviewer 2: if possible maybe have some for of emphasis here like bolding and caps Lol or # heading, not sure whats the most appropriate

Reviewer 2: Maybe also emphasize that anyone who has an osu! account can comment (this includes players and all)

Reviewer 3: ```suggestion
**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to the whole community. Know some banger maps from 2019 or a mapper left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open to _everyone_ to participate in, so let your positivity be heard!
```

> Writing comments will be open until the results of MCA are posted.

Reviewer 2: ```suggestion
Writing comments will be open **until** the results of MCA are posted.
```

All of these statistics and features can be found on the [AYIM Website](ayim.corsace.io)

## Mapper's Choice Awards 2019

MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards. 

> Eligible users are those who ranked a beatmap or storyboard in 2019.

Reviewer 2: Probably good to clarify that GDers are also eligible as well

MCA has three stages: Nomination, voting, and results.
- **Nomination:** Eligible users can submit any maps/storyboards or users from 2019 they feel are deserving of recognition in their respective categories.
- **Voting:** Eligible users can vote on the pool of submitted nominations.

> - **Results:** Will be based off the results of voting and feature a livestream discussing each of the top winners. 

Reviewer 2: May be a good idea to show this image somewhere? https://cdn.discordapp.com/attachments/789808737594245141/853671808255000636/unknown.png just so its more obvious as to how they can nominate/vote in storyboard as well

Author: Is there a version that would better fit for the newspost of these

This event will give perspective on what the community of that time finds to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019!

> Nomination and voting will be happening through the [MCA Website](mca.corsace.io) so be sure to check it out.

Reviewer 4: ```suggestion
Nomination and voting will be happening through the [MCA Website](mca.corsace.io), so be sure to check it out.
```

Nomination and voting will be happening through the [MCA Website](mca.corsace.io), so be sure to check it out.

> 
> 

Reviewer: you need to remove an extra blank line here
```suggestion

```

> We hope everyone can enjoy checking out MCA and AYIM as the events go on! Thank you to everyone who has made these possible, especially our main organizer [VINXIS](https://osu.ppy.sh/users/4323406). We look forwrard to seeing everyone's votes and comments for 2019.

Reviewer 3: ```suggestion
We hope everyone can enjoy checking out MCA and AYIM as the events go on! Thank you to everyone who has made these possible, especially our main organizer [VINXIS](https://osu.ppy.sh/users/4323406). We look forward to seeing everyone's votes and comments for 2019.
```

---
layout: post
title: "Mapper's Choice Awards and A Year In Mapping 2019"
date: 2021-06-14 00:00:00 +0000
---

Map connoisseurs rejoice, because the *Mappers' Choice Awards* AND *A Year In Mapping* (AYIM) are back and again and rearing to catch up with the scene!

It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later on in the year.

With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.

## Schedule for MCA and AYIM 2019

| Event | Timestamp (UTC) |
| --: | :-- |
| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
| Results Stream | 2021-07-10 / 2021-07-11 |
| Results posted to Website | 2021-07-12 00:00 |

## A Year In Mapping 2019

> AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and many, many more records than you can think of.

Reviewer 5: maybe explain a little bit here about what AYIM is for people who don't know? link some past contest results briefly, explain briefly how the voting works, so on, currently this doesn't tell anyone all that much of import

remember these contests have been dead for 2 years, that's an eternity for a lot of people who play

**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to *everyone*. Know some banger maps from 2019 or a mapper who left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open for everyone to participate in, so let your positivity be heard!

Writing comments will be open **until** the results of MCA are posted.

All of these statistics and features can be found on the [AYIM Website](https://ayim.corsace.io)

## Mapper's Choice Awards 2019

> MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards throughout multiple different categories. 

Reviewer 5: same deal here too, MCA is *huge* historically and you should at least cover why it was such a big deal and why people might want to care about it

---
layout: post
title: Mapper's Choice Awards and A Year In Mapping 2019
date: 2021-06-14 00:00:00 +0000
---

This time, we're going to experience celebrating mapping back-to-back for both 2019 and 2020!

This June and July will be Mappers' Choice Awards *(MCA)* and A Year in Mapping *(AYIM)* for the 2019 season! Soon after that, we will be hosting the same events but for 2020!

> It's been a while, but we're also back with an epic pair of websites as well to allow for a better experience than ever!

Reviewer 5: ```suggestion
With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.
```

> For now, we're all about celebrating 2019!

Reviewer 5: ```suggestion
```

---
layout: post
title: "Mapper's Choice Awards and A Year In Mapping 2019"
date: 2021-06-14 00:00:00 +0000
---

> Map connoisseurs rejoice, because the *Mappers' Choice Awards* AND *A Year In Mapping* (AYIM) are back and again and rearing to catch up with the scene!

Reviewer 4: ```suggestion
Map connoisseurs rejoice, because the *Mappers' Choice Awards* (MCA) AND *A Year In Mapping* (AYIM) are back again and raring to catch up with the scene!
```

![](/wiki/shared/news/2021-06-13-mca-ayim-2019/banner.png)

> It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later on in the year.

Reviewer 4: ```suggestion
It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later in the year.
```

With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.

> 

Reviewer 4: ```suggestion
```

Writing comments will be open **until** the results of MCA are posted.

All of these statistics and features can be found on the [AYIM Website](https://ayim.corsace.io)

## Mapper's Choice Awards 2019

MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards throughout multiple different categories. 

Eligible users are those who ranked a beatmap, guest difficulty, or storyboard in 2019.

MCA has three stages: Nomination, voting, and results.

- **Nomination:** Eligible users can submit any maps/storyboards or users from 2019 they feel are deserving of recognition in their respective categories.
- **Voting:** Eligible users can vote on the pool of submitted nominations.
- **Results:** Will be based off the results of voting and feature a livestream discussing each of the top winners. 

Those who win regular awards on their maps or storyboards in the results will be given 2 months of supporter, and grand prize winners will receive 6 months of supporter.

> Overall, this event will give perspective on what the community of that time finds to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019! 

Reviewer 4: ```suggestion
Overall, this event will give perspective on what the community of that time found to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019! 
```

Author: This event is being hosted _now_ reflecting on 2019, so it'd be present tense. People's impressions and what they remember most inevitably changes over time and results now would not be the same as then.

With all-new redesigned websites for these two titanic events, it's never been easier to get involved and have your say on what beatmaps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.

## Schedule for MCA and AYIM 2019

| Event | Timestamp (UTC) |
| --: | :-- |
| Nomination stage | 2021-06-15 00:00 to 2021-06-22 00:00 |
| Voting stage | 2021-06-24 00:00 to 2021-07-07 00:00 |
| Results Stream | 2021-07-10 / 2021-07-11 |
| Results posted to Website | 2021-07-12 00:00 |

## A Year In Mapping 2019

AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. 

### Statistics

As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and many, many more records than you can think of. You like interesting fun facts, trivia, fancy numbers? AYIM has all of those covered for the beatmaps and mappers ranked in 2019.

### Commenting on 2019 mappers

> Before, organizers were responsible for writing comments on each mapper ranked that year. These comments would sum up their achievements and place in the 2019 community. Everyone contributes their part, and AYIM seeks to help put that out there for others to see, learn, and appreciate. For an idea of what that has looked like in the past, check out the [AYIM 2018 Forum thread](https://osu.ppy.sh/community/forums/topics/966004).

Reviewer 4: sounds like this is saying that we rank mappers, which might sound off

```suggestion
Before, organizers were responsible for writing comments on each mapper who ranked maps that year. These comments would sum up their achievements and place in the 2019 community. Everyone contributes their part, and AYIM seeks to help put that out there for others to see, learn, and appreciate. For an idea of what that has looked like in the past, check out the [AYIM 2018 Forum thread](https://osu.ppy.sh/community/forums/topics/966004).
```
